modules/installation-overview.adoc

```diff
@@ -145,12 +164,16 @@ installation program to generate the assets that you require to provision the
 cluster infrastructure, create the cluster infrastructure, and then deploy
 the cluster to the infrastructure that you provided.
 
-If you do not use infrastructure that the installer provisioned, you must manage
+If you do not use infrastructure that the installation program provisioned, you must manage
 and maintain the cluster resources yourself, including:
 
 * The control plane and compute machines that make up the cluster
 * Load balancers
 * Cluster networking, including the DNS records and required subnets
+* Storage for the cluster infrastructure and applications
+
+If your cluster uses user-provisioned infrastructure, you have the option of
+adding RHEL worker nodes to your cluster.
```

Comment:
Can't you do this with an installer-provisioned cluster too?

Reply:
@mtnbikenc, in the [BYO RHEL compute](https://docs.openshift.com/container-platform/4.1/machine_management/adding-rhel-compute.html) topic, we say: 
> In OpenShift Container Platform, you can add Red Hat Enterprise Linux (RHEL) compute, or worker, machines to a user-provisioned infrastructure cluster. You can use RHEL as the operating system on only compute machines.

Is that right?